apps/src/aichat/views/modelCustomization/constants.ts

```diff
@@ -53,3 +53,13 @@ export const DEFAULT_LEVEL_AICHAT_SETTINGS: LevelAichatSettings = {
   visibilities: DEFAULT_VISIBILITIES,
   hidePresentationPanel: false,
 };
+
+export const READABLE_AI_CUSTOMIZATIONS: {
```

Comment:
super nit: what do you think about naming this something like `AI_CUSTOMIZATIONS_LABELS` or `TRANSLATED_AI_LABELS` since these will be eventually coming from translated strings?

Reply:
Much better, thanks!